frontend/packages/core-auth/src/components/AuthWrapper.tsx

```diff
@@ -45,6 +48,13 @@ const AuthWrapper: React.FC<AuthWrapperProps> = ({
     browser,
   );
 
+  // store tokenresponse in session storage
+  useEffect(() => {
+    if (tokenResponse?.accessToken) {
+      sessionStorage.setItem(injectToken, JSON.stringify(tokenResponse));
+    }
+  }, [tokenResponse?.accessToken]);
```

Comment:
This isn't always `accessToken`. It depends on `injectToken`. I would take this block outside of it's useEffect and whenever we have `accessToken` in useEffect deps,  we just use `token`.
```
const token = (() => {
      switch (injectToken) {
        case 'access_token':
          return tokenResponse?.accessToken ?? '';
        case 'id_token':
          return tokenResponse?.idToken ?? '';
        default:
          return '';
      }
    })();
  ```